src/it/java/org/checkstyle/plugins/sonar/CheckJarTest.java

```diff
@@ -26,7 +26,7 @@
 import org.junit.Test;
 
 public class CheckJarTest {
-    private static final String VERSION = "4.29";
+    private static final String VERSION = "4.30";
```

Comment:
very unfortunate copy-paste ..... do we know how to avoid this version definition here ?
can we parse pom to get this value ? or .... ?

Reply:
I created #288. I actually believe for this certain use case in the test, we could simply perform a file matching in the folder for the pattern. In CI there should really never be more than one jar file matching the name.

Reply:
Good idea